dissect/target/tools/utils.py

```diff
@@ -261,3 +263,24 @@ def wrapper(*args, **kwargs):
             raise
 
     return wrapper
+
+
+def args_to_uri(targets: list[str], loader_name: str, rest: list[str]) -> list[str]:
+    """Converts argument-style ``-L`` to URI-style.
+
+    Turns:
+        ``target-query /evtxs/* -L log --log-hint="evtx" -f evtx``
+    into:
+        ``target-query "log:///evtxs/*?hint=evtx" -f evtx``
+
+    For classes providing a __args__ dict with argparse values.
+    """
+    loader = LOADERS_BY_SCHEME.get(loader_name, None)
```

Comment:
Can you add a unit test for a non-existent loader?

Reply:
Sure...